### ISSUE TITLE
Build a model for recogizing brazilian coffee scenes

### ISSUE THREAD

**Issue by harpomaxx** [harpomaxx/image-processing#6]
The idea is to configure a neural network architecture for detecting coffee scenes using the 
[Brazilian Coffee Scenes Dataset](http://patreo.dcc.ufmg.br/2017/11/12/brazilian-coffee-scenes-d).

## Description

The whole image set of each country was partitioned into multiple tiles of 64 x 64 pixels. The complete dataset contains 2876 images separated into 4 folds with 600 images each and a 5th fold with only 476 images.

## Methodology 

The first 4 folds will be used for training while the 5th fold will be used for testing. On the 2400 images (4x600) of the training we can use different validation schemes:

1. k-fold Cross-Validation (using the preselected 4 folds o building new ones)
2. Monte Carlo Cross-Validation (shuffling the training dataset, and selecting 70% for training and %30 for validation. Repeating the process ~30 times

**IMPORTANT:**
Avoid looking at the testing set until you have performed all possible validation on the training set. You can also leave aside a minimal number of data from the training set to simulate a new (testing set)

 ## Articles:
 
[1] [Machine Learning Experimental Design 101](https://harpomaxx.github.io/post/experimental-design/) An article discussing the basis of model evaluation.
[2]  [Machine Learning Experimental Design 102](https://harpomaxx.github.io/post/experimental-design-102/). Considerations when dealing with features transformation during evaluations.
[3] Book ISLRV2 chapter 5 (cross-validation)
